praw/config.py

```diff
@@ -33,7 +33,7 @@ class Config:
     def _config_boolean(item):
         if isinstance(item, bool):
             return item
-        return item.lower() in {"1", "yes", "true", "on"}
+        return str(item).lower() in {"1", "yes", "true", "on"}
```

Comment:
We we mentioned before, I don't think this is neccessary. Config items always come in as a string.

Reply:
No, what about custom config options passed through Reddit(..)?